### ISSUE TITLE
Feature Request for Map Array Field

### ISSUE THREAD

**Issue by jeemercado** [firecmsco/firecms#587]
Is it possible to add an option to be able to hide/remove the "remove" button in this field?
![image](https://github.com/firecmsco/firecms/assets/53922337/efca8d4c-3074-4a77-a06b-1d1efa111188)

**Comment by fgatti675** [comment]
so you would be able to add but not delete?

**Comment by jeemercado** [comment]
Yup. It would be nice to be able to disable the other buttons as well like the moving and copying 🙂
